### PR TITLE
Fix link for RRM module homepage.

### PR DESCRIPTION
**Pull request by ankitrox**
## Summary

Addresses issue:

- #9144 

## PR Author Checklist

- [x] My code is tested and passes existing unit tests.
- [x] My code has an appropriate set of unit tests which all pass.
- [ ] My code is backward-compatible with WordPress 5.2 and PHP 7.4.
- [x] My code follows the [WordPress](https://make.wordpress.org/core/handbook/best-practices/coding-standards/) coding standards.
- [x] My code has proper inline documentation.
- [x] I have added a QA Brief on the issue linked above.
- [x] I have signed the Contributor License Agreement (see <https://cla.developers.google.com/>).

---------------

_Do not alter or remove anything below. The following sections will be managed by moderators only._

## Code Reviewer Checklist

- [x] Run the code.
- [x] Ensure the acceptance criteria are satisfied.
- [x] Reassess the implementation with the IB.
- [x] Ensure no unrelated changes are included.
- [ ] Ensure CI checks pass.
- [ ] Check Storybook where applicable.
- [x] Ensure there is a QA Brief.

## Merge Reviewer Checklist

- [ ] Ensure the PR has the correct target branch.
- [ ] Double-check that the PR is okay to be merged.
- [ ] Ensure the corresponding issue has a ZenHub release assigned.
- [ ] Add a changelog message to the issue.

### CODE REVIEW

File: includes/Modules/Reader_Revenue_Manager.php [hunk 1]
```diff
@@ -240,7 +240,7 @@ protected function setup_info() {
 			'name'        => _x( 'Reader Revenue Manager', 'Service name', 'google-site-kit' ),
 			'description' => __( 'Reader Revenue Manager helps publishers grow, retain, and engage their audiences, creating new revenue opportunities', 'google-site-kit' ),
 			'order'       => 5,
-			'homepage'    => __( 'https://readerrevenue.withgoogle.com/', 'google-site-kit' ),
+			'homepage'    => __( 'https://publishercenter.google.com', 'google-site-kit' ),
```
Comment (tofumatt) [inline includes/Modules/Reader_Revenue_Manager.php:243]: Why is this URL localised? 🤔 

We can't expect (community) localisers to know the right URL to send users to, so let's remove this.

```suggestion
			'homepage'    => 'https://publishercenter.google.com',
```

**Review by tofumatt — CHANGES_REQUESTED** [review]
Let's update the URL here, as discussed on our engineering call.

File: includes/Modules/Reader_Revenue_Manager.php (hunk 1 above)
Comment (nfmohit) [reply includes/Modules/Reader_Revenue_Manager.php:243]: This is actually existent in all modules. Should we maybe create an issue to remove all of them as well?

Comment (nfmohit) [reply includes/Modules/Reader_Revenue_Manager.php]: As this is a very small change, I've made this myself. I've also opened [#9175](https://github.com/google/site-kit-wp/issues/9175) to do this for other modules.

**Review by nfmohit — APPROVED** [review]
LGTM! ✅ 

Over to you @tofumatt to approve the PR.

**Review by tofumatt — APPROVED** [review]

**Pull request merged**